clients/src/main/java/org/apache/kafka/clients/admin/internals/MetadataOperationContext.java

```diff
@@ -82,6 +82,7 @@ public long deadline() {
 
     public static void handleMetadataErrors(MetadataResponse response) {
         for (TopicMetadata tm : response.topicMetadata()) {
+            if (shouldRefreshMetadata(tm.error())) throw tm.error().exception();
```

Comment:
nit: a stylistic observation is that this call is exactly the same as the call below. Yet it's written differently. 

Styles differ, usually not too much, from module to module. I think it's good to keep existing styles to help with readability and when the changes don't require a greater change. 
 
```suggestion
            if (shouldRefreshMetadata(tm.error())) { 
                throw tm.error().exception();
            }
```